packages/wast-parser/test/tokenizer/number-literals/+42/expected.json

```diff
@@ -0,0 +1 @@
+[{"type":"number","value":"+42","loc":{"start":{"line":1,"column":1}}}]
```

Comment:
To improve the readability, could we please use `JSON.stringify(x, null, 2)` here?

Reply:
Oh I didn't know that it supports formatting :sunglasses: Thanks!